dbt_meshify/dbt_projects.py

```diff
@@ -285,12 +342,52 @@ def split(
 
         return subproject
 
-    def initialize(self, target_directory: os.PathLike):
-        """Initialize this subproject as a full dbt project at the provided `target_directory`."""
+    def write_project_file(self):
+        """
+        Writes the dbt_project.yml file for the subproject in the specified subdirectory
+        """
+        contents = self.project.to_dict()
+        # was gettinga  weird serialization error from ruamel on this value
+        # it's been deprecated, so no reason to keep it
+        contents.pop("version")
+        # this one appears in the project yml, but i don't think it should be written
+        contents.pop("query-comment")
+        contents = filter_empty_dict_items(contents)
+        self.file_manager.write_file(Path("dbt_project.yml"), contents)
```

Comment:
may not want the file manager in this class -- to date, it's only in the `DbtMeshConstructor` class that's operating on resource files.

Reply:
Yeah, this feels like an odd level of abstraction. My mind goes to some other class that operates on `DbtProject`s to write files. Open to other approaches, though!